carapp/schemes.py

```diff
@@ -0,0 +1,141 @@
+from flask import jsonify
+from carapp import app
+from marshmallow import Schema, fields, validate
+
+
+"""
+Function to handle silent validation error.
+Get json with error instead
+"""
+
+
+@app.errorhandler(422)
+@app.errorhandler(400)
+def handle_error(err):
+    headers = err.data.get("headers", None)
+    messages = err.data.get("messages", ["Invalid request."])
+    if headers:
+        return jsonify({"errors": messages}), err.code, headers
+    else:
+        return jsonify({"errors": messages}), err.code
```

Comment:
This line is redundant. If `headers == {}` it should be ok to pass it blindly. So only line #18 is required.

Reply:
Ok, sure :)